clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -22,13 +22,19 @@ trait ClioParsers {
   implicit def enumEntryParser[T <: EnumEntry: Enum](
     implicit c: ClassTag[T]
   ): ArgParser[T] = {
-    ArgParser.instance[T]("entry") { entry =>
-      val value = implicitly[Enum[T]].withNameOption(entry)
-      Either.cond(
-        value.isDefined,
-        value.get,
-        s"Unknown enum value $entry for type ${c.runtimeClass.getName} "
-      )
+    val enum = implicitly[Enum[T]]
+    val enumName = c.runtimeClass.getName
+    val entryMap = enum.namesToValuesMap
+    val entryString = entryMap.keys.mkString("|")
```

Comment:
Can you use a comma list here instead? I think that would be easier to read than names separated by pipes.